src/firefly/js/tables/ui/BasicTableView.jsx

```diff
@@ -147,7 +147,10 @@ const BasicTableViewInternal = React.memo((props) => {
     };
 
     return (
-        <Box tabIndex='-1' onKeyDown={onKeyDown} sx={{lineHeight:1, flexGrow:1, minHeight:0, minWidth:0}}>
+        <Box tabIndex='-1' onKeyDown={onKeyDown} sx={(theme) => ({
+            lineHeight:1, flexGrow:1, minHeight:0, minWidth:0,
+            color: theme?.palette?.text?.secondary //otherwise parent's font color bleeds (which is not same in dark mode)
```

Comment:
instead of `(theme) =>`, you can use short-hand `text.secondary` syntax.

Reply:
Thanks - I just found out sx theme function was actually not responding to theme change in popups. Replacing with this string shorthand fixes it.